### ISSUE TITLE
doesn't work on svelte html comments with `eslint-plugin-svelte`

### ISSUE THREAD

**Issue by DetachHead** [mysticatea/eslint-plugin-eslint-comments#74]
i'm not sure if this is an issue with this plugin or `eslint-plugin-svelte`, so i'm raising it on both projects

from https://github.com/ota-meshi/eslint-plugin-svelte/issues/328:
> ### Before You File a Bug Report Please Confirm You Have Done The Following...
> 
>     * [x]  I have tried restarting my IDE and the issue persists.
> 
>     * [x]  I have updated to the latest version of the packages.
> 
> 
> ### What version of ESLint are you using?
> 
> 8.30.0
> ### What version of `eslint-plugin-svelte` are you using?
> 
> 2.14.1
> ### What did you do?
> Configuration
> 
> ```svelte
> <script>
>     // error: eslint-comments/require-description
>     //eslint-disable-next-line
>     console.log(1)
> </script>
> 
> 
> 
> <input value="asdf">
> ```
> 
> ### What did you expect to happen?
> 
> `eslint-comments/require-description` error on the html comment
> ### What actually happened?
> 
> no error on the html comment
> ### Link to **GitHub Repo** with Minimal Reproducible Example
> 
> https://github.com/DetachHead/eslint-plugin-svelte-issue/tree/d0eb35b9d72cd26928488f7bbe8cb0ee0f0d1222
> ### Additional comments
> 
> i'm not sure whether this is an issue with this plugin or `eslint-plugin-eslint-comments`, so i'm raising it in both projects

**Comment by DetachHead** [comment]
see https://github.com/ota-meshi/eslint-plugin-svelte/issues/328#issuecomment-1357742731
